modules/renter/repair.go

```diff
@@ -491,6 +491,7 @@ func (r *Renter) threadedRepairLoop() {
 		resultChan:       make(chan finishedUpload),
 	}
 	for {
+		// TODO: Handle error.
 		if r.tg.Add() != nil {
 			return
 		}
```

Comment:
Removed the TODO. @lukechampine makes sense that no specific handling is necessary. It seemed odd without knowing the full context.